students/10th/gonholee/westagram/posts/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls    import path
+from .              import views
+
+urlpatterns = [
+    path('/', views.PostRegister.as_view()),
+    path('/comment-regist',views.CommentRegister.as_view()),
```

Comment:
comment-register 로 하실려고 한것같은데 RESTful 엔드포인트 방식에서는 resource를 명사로 표현하고 행위를 HTTP method로 표현하는데 그에 맞게 수정 부탁 드립니다. 
```suggestion
    path('/comment',views.CommentRegister.as_view()),
```